lib/instana/instrumentation/sequel.rb

```diff
@@ -0,0 +1,41 @@
+# (c) Copyright IBM Corp. 2024
+
+module Instana
+  module Instrumentation
+    module Sequel
+      IGNORED_SQL = %w[BEGIN COMMIT SET].freeze
+      SANITIZE_REGEXP = /('[\s\S][^']*'|\d*\.\d+|\d+|NULL)/i.freeze
```

Comment:
Why `[\s\S]` over just `.`? Do we expect a linebreak here?
In general why is it needed? Why not just the `[^']*'`?
Why does `NULL` need to be sanitized?

Reply:
This regex is present in active record. So i copied the same. We sanitise sql so that the ppl viewing the logs might not get the information like PII etc etc. So NULL can be used to filter data. Hence i think we are sanitising NULL.

Reply:
I am familiar with the purpose of sanitation. My point is that the string `NULL` alone is not a secret, so without further reason, it makes no sense to sanitize it.
And I have also noticed, that active record has this pattern, and I don't think we should randomly copy code without understanding it.